pom.xml

```diff
@@ -26,6 +27,16 @@
                 <artifactId>lombok</artifactId>
                 <version>1.18.34</version>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot</artifactId>
+                <version>3.3.2</version>
```

Comment:
Some versions generally need to be linked together, just like these two

In this case we use properties

Up above the dependencies you can declare some properties:

<properties>
    <spring.version>3.3.2</spring.version>
</properties>

and then use a placeholder here

<version>${spring.version}</version>